### PR TITLE
Add support for building multiple ditaa diagrams

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

File: docs/ditaa.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const glob = require("glob")
```
Comment (tylerbutler) [inline docs/ditaa.js:6]:
```suggestion
const glob = require("glob");
```

File: docs/ditaa.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const glob = require("glob");
+const shell = require("shelljs");
+
+const args = process.argv.slice(2);
+const glob_arg = args[0];
+
+try {
+    const files = glob.sync(glob_arg);
+    console.log(files);
```
Comment (micahgodbolt) [inline docs/ditaa.js:14]: need this log?

**Comment by micahgodbolt** [comment]
oh cool....does this approach not require java?

**Review by micahgodbolt — APPROVED** [review]
other than console.log, looks good

**Comment by tylerbutler** [comment]
> oh cool....does this approach not require java?

Alas, no, it still does. It was easy to add to the build pipeline, but might not be the best long-term solution.

File: docs/ditaa.js (hunk 2 above)
Comment (tylerbutler) [inline docs/ditaa.js:14]:
```suggestion
```

**Comment by curtisman** [comment]
Closing and reopening to trigger build

**Comment by tylerbutler** [comment]
@micahgodbolt There is no longer a Java dependency. :)

**Pull request merged**